bioptim/optimization/optimal_control_program.py

```diff
@@ -403,7 +411,7 @@ def __init__(
         # Prepare phase transitions (Reminder, it is important that parameters are declared before,
         # otherwise they will erase the phase_transitions)
         self.phase_transitions = phase_transitions.prepare_phase_transitions(self)
-
+        self.multinode_constraints = multinode_constraints.prepare_multinode_constraints(self)
```

Comment:
If it is mandatory to leave it here, that is allright, otherwise, please move it down with "self.update_contraints" stuff

Reply:
done.

Reply:
Please revert!

Reply:
reverted